corehq/apps/case_search/models.py

```diff
@@ -19,7 +19,7 @@ class Meta:
         primary_key=True
     )
     enabled = models.BooleanField(blank=False, null=False, default=False)
-    config = JSONField(default={})
+    config = JSONField(default=dict())
```

Comment:
I think this is still wrong - should be `dict` the function, not the output of the function